### PR TITLE
chore: create Tooltip component UXDS-187

### PR DESCRIPTION
**Pull request by owolfhu1**

### CODE REVIEW

File: packages/tooltip/src/lib/Tooltip.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React, { forwardRef, ReactElement } from 'react';
+import { Tooltip as MuiTooltip, TooltipProps as MuiTooltipProps } from '@mui/material';
+
+export type TooltipProps = MuiTooltipProps;
+
+export const Tooltip = forwardRef<ReactElement, TooltipProps>((props: TooltipProps, forwardedRef) => {
+  let { children, arrow, ...rest } = props;
+
+  if (arrow !== false) {
```
Comment (JoeSpanbauer) [inline packages/tooltip/src/lib/Tooltip.tsx:9]: This seems to be a little backwards.
`<Thing />` - Shows the arrow
`<Thing arrow />` - Shows the arrow
`<Thing arrow={false} />` - Doesn't show the arrow

Is the intention to only ever show the arrow for our tooltips?

Comment (owolfhu1) [reply packages/tooltip/src/lib/Tooltip.tsx:9]: Tyler requested we make this property default to true, out of the box it defaults to false.

Comment (owolfhu1) [reply packages/tooltip/src/lib/Tooltip.tsx]: This has been updated to set arrow to true internally and disallow the user from modifying it.

Comment (owolfhu1) [reply packages/tooltip/src/lib/Tooltip.tsx]: Good catch, omitted

File: packages/tooltip/src/lib/Tooltip.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,81 @@
+// Each exported component in the package should have its own stories file
+
+import type { StoryObj } from '@storybook/react';
+import { Tooltip, TooltipProps } from './Tooltip';
+
+// TODO: use our components when forwardRef is being applied correctly
```
Comment (JoeSpanbauer) [inline packages/tooltip/src/lib/Tooltip.stories.tsx:6]: Can we get rid of this TODO?

**Review by JoeSpanbauer — APPROVED** [review]
Only one question about a TODO comment. If it's still necessary, then good to go. Though, I would remove it before merging, if we don't need it.

File: packages/tooltip/src/lib/Tooltip.stories.tsx (hunk 2 above)
Comment (owolfhu1) [reply packages/tooltip/src/lib/Tooltip.stories.tsx:6]: I think we should try to use our own components in our docs so I we should probably keep this.

**Pull request merged**